pysat/model_utils.py

```diff
@@ -525,9 +532,9 @@ def extract_modelled_observations(inst=None, model=None, inst_name=[],
                         idims = len(inst.data.coords)
                         idim_names = inst.data.coords.keys()[1:]
 
-                        # Find relevent dimensions for cycling
+                        # Find relevent dimensions for cycling and slicing
```

Comment:
I'm really not up to speed with what is happening here, so this comment may not make sense. The instrument object support integer indexing and slicing at the Object level inst['var', idx, idy, idx, ...] over however many coordinates there are. xarray doesn't natively support it but I put in a workaround.

Reply:
Yeah, I don't think your statement is relevant here.  Could be wrong though.  Getting the right slice of data based on the possible overlap of model and instrument was a bit tricky.

Reply:
Fair enough. I hear ya.